Example/Sources/ConversationViewController.swift

```diff
@@ -98,8 +109,12 @@ class ConversationViewController: MessagesViewController {
                 print("# Selected")
             },
             .flexibleSpace,
-            makeButton(named: "ic_library").onTextViewDidChange { button, textView in
-                button.isEnabled = textView.text.isEmpty
+            makeButton(named: "ic_library").onSelected {
+                $0.tintColor = UIColor(red: 15/255, green: 135/255, blue: 255/255, alpha: 1.0)
```

Comment:
How do we feel about pulling colors into their own `Styles` file or something?

Reply:
This is the Example

Reply:
Derp... carry on.